### DISCUSSION TITLE
（求助）紧急！！！求配队策略

### DISCUSSION THREAD

**Discussion by Aries-Woof** [share121/inter-knot#2898] [其他]
目前0命莱卡恩，1命猫又，0命格莉丝，其他四星齐了，本，安比，苍角，妮可都有三四命<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by Aries-Woof** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/a63b1fdb4c2a4562c7d291b230978f0a_944582820103199731.png" alt="丽都漫步-莱卡恩" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/a63b1fdb4c2a4562c7d291b230978f0a_944582820103199731.png" alt="丽都漫步-莱卡恩" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/a63b1fdb4c2a4562c7d291b230978f0a_944582820103199731.png" alt="丽都漫步-莱卡恩" width="75" >
为什么我没有个S主C啊啊啊啊啊啊啊啊啊啊玛卡巴卡也很好啊啊啊啊啊啊啊啊啊

**Comment by 600310** [comment]
去玩电疗队，安东主c

**Comment by kkxx-kkxx** [comment]
格莉丝 安比 妮可（丽娜）异常队
猫又 派派 露西 物理队
（不是很懂，给个建议，出错轻喷）
<img src="https://bbs-static.miyoushe.com/static/2023/07/12/06e21f5dc06d93f9e69144c621114dd3_6443285262123990781.png" alt="帕姆_不可以" width="75" >

**Comment by chihung3927** [comment]
莱卡恩 妮可 苍角([主C](https://www.bilibili.com/video/BV1aS421X7Gp/?spm_id_from=333.337.search-card.all.click))

**Reply by Aries-Woof** [reply]
啊？苍角当主C？！？！
他不是支援角色吗

**Reply by chihung3927** [reply]
详情请看超链接!

**Comment by yitan1741** [comment]
个人一队用猫又、派派、露西（物理队）还是很舒服的，猫又速切c，不站场，派派站场用转圈最后一个砸的无敌帧躲伤害，露西时不时切出来增伤。
如果怪的攻击欲望比较强，就猫又站场，打进失衡切派派转圈，失衡期间可以打两个强击，一个强击大概10万左右（第一个一般为7万左右，第二个因为减防可以来到10万）

个人二队用的是抹茶小蛋糕（可琳）c，暂时体验还是不错的，因为我的击破只有安比，并且有女仆长，所以体验不是很好（前期女仆长辅助确实有点拉胯），不过练到后期伤害还是很可观。但您有狼叔，个人觉得可以狼叔击破，苍角辅助（前期苍角辅助能力很强），这样编队buff一样可以吃满，击破效率也会很高，重点是狼叔要站场击破，可琳只有在怪物失衡时再切出打伤害，打高血量BOSS时，在可琳输出前可能要控制一下失衡值，让苍角的被动先叠满，吃完buff再去打进失衡。

格丽斯暂时并不建议您练，因为现阶段，异常紊乱队手法要求过高，因格丽斯和派派都需要一定的站场空间，且异常体系并不完善。

至于驱动盘，建议直接去搜攻略，因为每个队伍对一个角色的要求都不一样，就比如说猫又，站场建议四暴击二物伤，速切强击建议四物伤二暴击，或者四物伤二攻击（我记不住盘的名字，就直接说属性了）。

总的来说，新手期资源极其匮乏，要练先练主c，谁打伤害就练谁，把主c练起来以后，至少不会太刮。

个人玩法总结，经供参考，若能帮助到您实属万幸<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >
